packages/event-target/redis-event-target/src/index.ts

```diff
@@ -5,13 +5,20 @@ import { CustomEvent } from '@whatwg-node/events';
 export type CreateRedisEventTargetArgs = {
   publishClient: Redis | Cluster;
   subscribeClient: Redis | Cluster;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  serializeMessage?: (message: any) => string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  deserializeMessage?: (message: string) => any;
```

Comment:
do you think it would make sense to have these two properties under an object instead? That way we can just use JSON as the default option 🤔

```
serializer: {
  parse: ...,
  stringify: ...,
}
```